### PR TITLE
Light level polling command

### PR DESCRIPTION
**Pull request by crami**
This pull request implements a new command to get brightness and status of lamps and groups.

### CODE REVIEW

File: dali_mqtt_daemon.py [hunk 1]
```diff
@@ -375,6 +376,56 @@ def on_message_brightness_cmd(mqtt_client, data_object, msg):
...
 
+def on_message_brightness_get_cmd(mqtt_client, data_object, msg):
+    """Callback on MQTT brightness get command message."""
+    logger.debug("Brightness Get Command on %s: %s", msg.topic, msg.payload)
+    light = re.search(
+        MQTT_BRIGHTNESS_GET_COMMAND_TOPIC.format(data_object["base_topic"], "(.+?)"),
+        msg.topic,
+    ).group(1)
+    try:
+        if 'group_' in light:
```
Comment (dgomes) [inline dali_mqtt_daemon.py:387]: This block is the same as `on_message_brightness_cmd`

could you please refactor, so we don't repeat this code ?

File: dali_mqtt_daemon.py [hunk 2]
```diff
@@ -375,6 +376,56 @@ def on_message_brightness_cmd(mqtt_client, data_object, msg):
...
+                retain=False,
+            )
+            if level.value == 0:
+              lamp_state=MQTT_PAYLOAD_OFF
+            else:
+              lamp_state=MQTT_PAYLOAD_ON
+                
+            mqtt_client.publish(
+                MQTT_STATE_TOPIC.format(data_object["base_topic"], light),
+                lamp_state,
```
Comment (dgomes) [inline dali_mqtt_daemon.py:415]: Can be made inline:

`MQTT_PAYLOAD_ON if level.value != 0 else MQTT_PAYLOAD_OFF`

**Review by dgomes — COMMENTED** [review]
Wasn't the plan to get the status periodically? that is, without the need for an MQTT command ?

File: dali_mqtt_daemon.py [hunk 3]
```diff
@@ -320,6 +320,18 @@ def on_message_reinitialize_lamps_cmd(mqtt_client, data_object, msg):
...
+    if 'group_' in light:
+        """ Check if the comand is for a dali group """
+        group = int(re.search('group_(\d+)', light).group(1))
+        lamp_object=data_object["all_lamps"][group]
+    else:
+        """ The command is for a single lamp """
+        if light not in data_object["all_lamps"]:
+            raise KeyError
+        lamp_object = data_object["all_lamps"][light]
+    return(lamp_object)
```
Comment (dgomes) [inline dali_mqtt_daemon.py:333]:
```suggestion
    return lamp_object
```

**Comment by crami** [comment]
I'm quite happy now how this works. Can you please review again.

File: dali_mqtt_daemon.py [hunk 4]
```diff
@@ -375,6 +375,43 @@ def on_message_brightness_cmd(mqtt_client, data_object, msg):
...
+    """Callback on MQTT brightness get command message."""
+    logger.debug("Brightness Get Command on %s: %s", msg.topic, msg.payload)
+    light = re.search(
+        MQTT_BRIGHTNESS_GET_COMMAND_TOPIC.format(data_object["base_topic"], "(.+?)"),
+        msg.topic,
+    ).group(1)
+    try:
+        lamp_object = get_lamp_object(data_object, light)
+          
+        level = None
```
Comment (dgomes) [inline dali_mqtt_daemon.py:388]: not required

**Review by dgomes — CHANGES_REQUESTED** [review]
What about the idea of making this periodic ?

File: dali_mqtt_daemon.py (hunk 4 above)
Comment (crami) [reply dali_mqtt_daemon.py]: removed that line

**Comment by crami** [comment]
Will think about making it periodic, still no clue how to get from main to this point. Think it should be a configurable option and this functionality should be in anyway.

**Comment by dgomes** [comment]
Thank you!

**Pull request merged**